commercetools-models/src/main/java/io/sphere/sdk/orders/commands/OrderFromCartInStoreCreateCommand.java

```diff
@@ -17,4 +17,8 @@ static OrderFromCartInStoreCreateCommand of(final String storeKey, final OrderFr
     static OrderFromCartInStoreCreateCommand of(final String storeKey, final Versioned<Cart> cart) {
         return new OrderFromCartInStoreCreateCommandImpl(storeKey, OrderFromCartDraft.of(cart));
     }
+
+    static OrderFromCartInStoreCreateCommand of(final String storeKey, final Cart cart) {
```

Comment:
not needed a Cart implements Versioned

Reply:
I'm replacing with ResourceIdentifier plus the Long version as mandatory field